### PR TITLE
Feature/ext osclib

### PR DESCRIPTION
**Pull request by cjbackhouse**
Pick up OscLib as an external product rather than duplicating it in sbncode.
    
The product has abbreviated of the names, which entails a lot of mechanical changes.

### CODE REVIEW

File: sbncode/CAFAna/Core/CMakeLists.txt [hunk 1]
```diff
@@ -3,8 +3,13 @@ include_directories( $ENV{IFDHC_FQ_DIR}/inc )
 # Allow "StandardRecord/" to find the right directory
 include_directories(..)
 
+include_directories($ENV{OSCLIB_INC})
+
 art_make(LIBRARY_NAME CAFAnaCore
          LIB_LIBRARIES StandardRecordProxy
+                       # TODO how am I supposed to link to an external library?
```
Comment (cjbackhouse) [inline sbncode/CAFAna/Core/CMakeLists.txt:10]: @wesketchum just ${OSCLIB} didn't work here like I expected. I think there a couple other instances of this pattern too.

**Comment by cjbackhouse** [comment]
I'd like to merge this before the sbnana split too. Otherwise I will have to recreate it over in sbnana. The osclib dependency that it adds will only be incurred by sbnana, which is one of the main points of doing the split.

**Comment by wesketchum** [comment]
I think that's not a problem at all: we'll do this merge before doing the plot.

**Comment by cjbackhouse** [comment]
Superseded by #78 and merged there

**Pull request closed without merging**